chroma-manager/tests/integration/core/remote_operations.py

```diff
@@ -264,6 +266,11 @@ def stop_target(self, fqdn, ha_label):
     def start_target(self, fqdn, ha_label):
         self._ssh_fqdn(fqdn, "chroma-agent start_target --ha %s" % ha_label)
 
+    def import_target(self, fqdn, type, path, pacemaker_ha_operation):
+        self._ssh_fqdn(fqdn, "chroma-agent import_target --device_type %s "
+                             "--path %s --pacemaker_ha_operation %s" %
+                              (type, path, pacemaker_ha_operation))
```

Comment:
Didn't we get rid of this `pacemaker_ha_operation` at some point?  I thought with the _device-scanner_ we didn't need this option any more.

Reply:
We removed it then reinstated it, it is still needed in the failover case where pool is not exported cleanly from a non-responsive node and the export `-f` flag is required.